README.md

```diff
@@ -151,6 +152,7 @@ The following table shows the supported combinations:
 
 | LiT               | LND          | Loop         | Faraday       | Pool         | Taproot Assets |
 |-------------------|--------------|--------------|---------------|--------------|----------------|
+| **v0.12.4-alpha** | v0.17.4-beta | v0.28.0-beta | v0.2.11-alpha | v0.6.4-beta  | v0.3.3-alpha   |
```

Comment:
Faraday version needs to be bumped here as well.

Reply:
Ah, good catch, thanks!